packages/lesswrong/lib/givingSeason.tsx

```diff
@@ -0,0 +1,157 @@
+import React, {
+  createContext,
+  Dispatch,
+  ReactNode,
+  SetStateAction,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+} from "react";
+import { useCurrentTime } from "./utils/timeUtil";
+import { useOnNavigate } from "@/components/hooks/useOnNavigate";
+import { useLocation } from "./routeUtil";
+import { useQuery } from "@apollo/client";
+import { isEAForum } from "./instanceSettings";
+import gql from "graphql-tag";
+
+export const GIVING_SEASON_INFO_HREF = "/posts/srZEX2r9upbwfnRKw"; // TODO
+export const ELECTION_INFO_HREF = "/posts/srZEX2r9upbwfnRKw"; // TODO
+export const ELECTION_DONATE_HREF = "/donation-portal";
+const ELECTION_TARGET_AMOUNT = 60000; // TODO
```

Comment:
Do you intend to add these before merging this?

Reply:
Yep